### ISSUE TITLE
FInals Submission questions

### ISSUE THREAD

**Issue by ZYN2122** [nus-cs2030/2223-s1#349]
Hey guys, does anyone know if we need to get the A grade for each question during finals to get marks? Or is it okay if we get for example F but they grade based on reading the code? Thanks!

**Comment by mohddanesh** [comment]
the grading is based on human grader reading the code. You won't get a grade from codecrunch so you don't even need to compile your code. the grading is purely based on your code logic

**Comment by ZYN2122** [comment]
Ahh okok so the profs will actually grade based on the logic rather than checking for compile problems?

**Comment by mohddanesh** [comment]
yup! he mentioned in the review video that you don't even need to compile your code during the exam. he also won't compile it when he grades it. he just wants to see your logic

**Comment by ZYN2122** [comment]
Oh we are also able to look at our past code right? local code and previous submitted codecrunch submissions for other labs etc

**Comment by qwqdrz** [comment]
> 

Yes we are allowed
